portfolio/src/main/webapp/index.html

```diff
@@ -5,63 +5,65 @@
-	   <button class="mdc-button mdc-button--outlined" onclick="interests()" style="color:white;">  
-	       <span class="mdc-button__ripple">Interests</span>
-	   </button>
+            <button class="mdc-button mdc-button--outlined" onclick="interests()" style="color:white;">  
+                <span class="mdc-button__ripple">Interests</span>
+            </button>
         </div>
-      </div>
-      <div id="basic-profile" style="display:inline-block;">
-        <img src="/images/PROFILE_PIC.JPG">
-      	<h1>Michaela Murray's Portfolio</h1>
-      	<div id="intro">
-      	  <ul style="list-style-type:none;">
-            <li>Stanford Class of 2022</li> 
-            <li>Major: Computer Science</li>
-            <li>STEP Intern @ Google</li>
-	  </ul>
         </div>
-      </div>
-      <div id="intro-button-two" style="display:inline-block;">
-	<div class="mdc-touch-target-wrapper" style="display:block;">
-	   <button class="mdc-button mdc-button--outlined" style="color:white;">  
-	      <span class="mdc-button__ripple">Blog</span>
-	   </button>
+        <div id="profile-wrapper" style="display:inline-block;">
+        <div id="basic-profile">
+            <div id="wrapper">
+                <img id="profile" src="/images/PROFILE_PIC.JPG">
+            </div>
+            <h1>Michaela Murray's Portfolio</h1>
+            <div id="intro">
+                <ul style="list-style-type: none;">
+                <li>Stanford Class of 2022</li> 
+                <li>Major: Computer Science</li>
+                <li>STEP Intern @ Google</li>
+                </ul>
+            </div>
+        </div>
+        </div>
+        <div id="intro-button-two" style="display: inline-block;">
+        <div class="mdc-touch-target-wrapper" style="display: block;">
+            <button class="mdc-button mdc-button--outlined" onclick="blog()" style="color:white;">  
+                <span class="mdc-button__ripple">Blog</span>
+            </button>
         </div>
         <div class="mdc-touch-target-wrapper" style="display:block;">
-          <form action="/pdf/Michaela_Murray_Resume.pdf" target="_blank" style="display:inline-block;">
+            <form action="/pdf/Michaela_Murray_Resume.pdf" target="_blank" style="display:inline-block;">
             <button class="mdc-button mdc-button--outlined" style="color:white;">  
-	       <span class="mdc-button__ripple">Resume</span> 
-	    </button>
-          </form>
-        </div> 
+                <span class="mdc-button__ripple">Resume</span>
+            </button>
+            </form>
+        </div>
         <div class="mdc-touch-target-wrapper" style="display:block;">
-      	  <button class="mdc-button mdc-button--outlined" onclick="contact()" style="color:white;">  
-	     <span class="mdc-button__ripple">Contact</span> 
-	  </button>
+            <button id="picture-button" class="mdc-button mdc-button--outlined" onclick="pictures()" style="color:white;">  
```

Comment:
nit - revert this change related to pictures and send in a separate PR.

Reply:
This change falls under the formatting PR-I will remove this from this PR thanks!